charts/stable/authentik/docs/how_to.md

```diff
@@ -0,0 +1,129 @@
+# How-To
+
+This is a quick how-to or setup guide to use Authentik with TrueNAS SCALE and setup a simple Proxy Provider with `traefik` using the Embedded Outpost to use as a Traefik `forwardauth`. This guide was created with Authentik `2022.9.0` and will be updated if things dramatically change.
+This can be applied to other systems but this specific guide has been tested and created on TrueNAS SCALE and isn't guaranteed to work with any other configs.
+
+## Requirements
+
+- Authentik TrueCharts Chart
+- Traefik Truecharts Chart
+
+## Prerequisites
+
+This guide assumes you're using Traefik as your Reverse Proxy / Ingress provider and have through the configuration listen in our [Quick-Start guides](https://truecharts.org/docs/manual/SCALE%20Apps/adding-letsencrypt) and/or the [Traefik documents](https://truecharts.org/docs/charts/stable/traefik/how-to/). Please ensure that you can access your domain properly with Ingress before attempting any further steps.
+
+## Authentik Chart Setup
+
+#### Note
```

Comment:
try using this instead, might look nicer.
more examples can be found here -> https://docusaurus.io/docs/next/markdown-features/admonitions

```
:::note

> The `Authentik` chart has many options, which may enhance or break your chart depending on your setup and are beyond the scope of this guide

:::
```

Reply:
Good call, I’m all about throwing info and not that good with the markdown